ophyd/_pyepics_shim.py

```diff
@@ -1,31 +1,74 @@
 import atexit
+import copy
 import ctypes
-import epics
+import functools
+import logging
 import queue
 import threading
 import warnings
 
-from epics import get_pv as _get_pv, caget, caget, caput
+from types import SimpleNamespace
+
+import epics
+from epics import get_pv as _get_pv, caget, caput, ca, dbr  # noqa
 
 try:
-    epics.ca.find_libca()
-except epics.ca.ChannelAccessException:
+    ca.find_libca()
+except ca.ChannelAccessException:
```

Comment:
This logic is to handle tho no-pyepics case so ophyd will import.  You can probably safety rip all of this out (and replace it by raising `ImportError` so the the fallback works).

Reply:
Thanks! Resolved in 55f1b8d.